src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -2745,11 +2765,24 @@ void CacheManager::MergeCFM(const policy_table::PolicyTable& new_pt,
 
 void CacheManager::InitBackupThread() {
   LOG4CXX_AUTO_TRACE(logger_);
+  sync_primitives::AutoLock lock(backuper_locker_);
   backuper_ = new BackgroundBackuper(this);
   backup_thread_ = threads::CreateThread("Backup thread", backuper_);
   backup_thread_->start();
 }
 
+void CacheManager::StopBackupTread() {
+  LOG4CXX_AUTO_TRACE(logger_);
+  sync_primitives::AutoLock lock(backuper_locker_);
+  if (NULL == backup_thread_) {
```

Comment:
@LitvinenkoIra let's use `nullptr`

Reply:
@LitvinenkoIra check if this condition should be checked with DCHECK